src/AppBundle/Controller/InfoMeetingController.php

```diff
@@ -0,0 +1,66 @@
+<?php
+
+namespace AppBundle\Controller;
+
+use AppBundle\Entity\Department;
+use AppBundle\Entity\InfoMeeting;
+use AppBundle\Form\Type\EditInfoMeetingType;
+use Symfony\Bundle\FrameworkBundle\Controller\Controller;
+use Symfony\Component\HttpFoundation\Request;
+
+class InfoMeetingController extends Controller
+{
+    public function showAction(Request $request, Department $department)
+    {
+        $infoMeeting = new InfoMeeting();
+        if ($department->getInfoMeeting() !== null) {
+            $infoMeeting = $department->getInfoMeeting();
+        }
+
+        $form = $this->createForm(EditInfoMeetingType::class);
```

Comment:
Hvis `$infoMeeting` legges inn i `createForm`, `$this->createForm(EditInfoMeetingType::class, $infoMeeting)`, så vil den auto mappe data fra formen til objektet. Så da trenger man ikke å gjøre
```
$data = $form->getData();
 
$infoMeeting->setTime($data['time']);
$infoMeeting->setDate($data['date']);
$infoMeeting->setRoom($data['room']);
$infoMeeting->setExtra($data['extra']);
```
(Men Department må fortsatt settes manuelt siden den ikke kommer fra formen)